api/src/main/java/com/epam/pipeline/controller/user/UserController.java

```diff
@@ -274,4 +292,48 @@ public Result deleteUser(@PathVariable Long id) {
     public Result<List<String>> findGroups(@RequestParam String prefix) {
         return Result.success(userApiService.findGroups(prefix));
     }
+
+    @PostMapping(value = "/group/block")
+    @ResponseBody
+    @ApiOperation(
+            value = "Creates the block status of a group.",
+            notes = "Creates the block status of a group. " +
+                    "If the group is blocked, none of its users can't access their accounts.",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+            value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+            })
+    public Result<GroupStatus> createGroupBlockingStatus(@PathVariable final String groupName,
+                                                         @PathVariable final Boolean blockStatus) {
+        return Result.success(userApiService.createGroupBlockingStatus(groupName, blockStatus));
+    }
+
+    @PutMapping(value = "/group/block")
+    @ResponseBody
+    @ApiOperation(
+            value = "Changes the block status of a group.",
+            notes = "Changes the block status of a group. " +
+                    "If the group is blocked, none of its users can't access their accounts.",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+            value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+            })
+    public Result<GroupStatus> updateGroupBlockingStatus(@PathVariable final String groupName,
+                                                         @PathVariable final Boolean blockStatus) {
+        return Result.success(userApiService.updateGroupBlockingStatus(groupName, blockStatus));
+    }
+
+    @DeleteMapping(value = "/group/block")
+    @ResponseBody
+    @ApiOperation(
+            value = "Removes the block status of a group.",
+            notes = "Removes the block status of a group.",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(
+            value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
+            })
+    public Result deleteGroupBlockingStatus(@PathVariable final String groupName) {
+        userApiService.deleteGroupBlockingStatus(groupName);
+        return Result.success(null);
```

Comment:
Could you return `userApiService.deleteGroupBlockingStatus(groupName)` instead of `null`?